tests/service/execute/test_dpsu.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+@pytest.mark.parametrize("project", [{"params": {"dataset_name": "PUMS", "budget": .2,
+                                      "query": "SELECT married, educ FROM PUMS.PUMS GROUP BY married, educ"},

```

Comment:
A local test, not in execution with private reader should run, and make sure that it is using dpsu if the right params are set

Reply:
Maybe adding an assert at the end would be enough to verify it.

assert PrivateReader()._get_reader is not PrivateReader.reader

And maybe a negative test if the reader is not a PandasReader